test/resources/layout-shared.css

```diff
@@ -10,3 +10,12 @@ body {
     background: beige;
     font-size: 14px;
 }
+
+.panel-playground, .panel-body-playground {
+    border: 1px solid navy;
+    border-radius: 4px;
+    padding: 10px;
+    background: gold;
+    font-size: 14px;
+    box-shadow: 0 5px 20px 0;
```

Comment:
Minor nit: box shadows should be defined with a color.  Without it, the playground box shadows are actually appearing themeable.